File: Android/Telehealth/Patient/TelehealthPatient/app/src/main/java/com/redimed/telehealth/patient/models/Patient.java
package com.redimed.telehealth.patient.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Lam on 10/12/2015.
 */
public class Patient {

    @SerializedName("UID")
    private String UID;

    @SerializedName("UserAccountID")
    private int UserAccountID;

    @SerializedName("FirstName")
    private String FirstName;

    @SerializedName("MiddleName")
    private String MiddleName;

    @SerializedName("LastName")
    private String LastName;

    @SerializedName("DOB")
    private String DOB;

    @SerializedName("Gender")
    private String Gender;

    @SerializedName("Address1")
    private String Address1;

    @SerializedName("Address2")
    private String Address2;

    @SerializedName("Suburb")
    private String Suburb;

    @SerializedName("PostCode")
    private String PostCode;

    @SerializedName("State")
    private String State;

    @SerializedName("Email")
    private String Email;

    @SerializedName("HomePhoneNumber")
    private String HomePhoneNumber;

    @SerializedName("Enable")
    private String Enable;

    @SerializedName("PhoneNumber")
    private String PhoneNumber;

    @SerializedName("UserAccount")
    private UserAccount userAccount;

    private String Limit;

    public String getUID() {
        return UID;
    }

    public void setUID(String UID) {
        this.UID = UID;
    }

    public int getUserAccountID() {
        return UserAccountID;
    }

    public void setUserAccountID(int userAccountID) {
        UserAccountID = userAccountID;
    }

    public String getFirstName() {
        return FirstName;
    }

    public void setFirstName(String firstName) {
        FirstName = firstName;
    }

    public String getMiddleName() {
        return MiddleName;
    }

    public void setMiddleName(String middleName) {
        MiddleName = middleName;
    }

    public String getLastName() {
        return LastName;
    }

    public void setLastName(String lastName) {
        LastName = lastName;
    }

    public String getDOB() {
        return DOB;
    }

    public void setDOB(String DOB) {
        this.DOB = DOB;
    }

    public String getGender() {
        return Gender;
    }

    public void setGender(String gender) {
        Gender = gender;
    }

    public String getAddress1() {
        return Address1;
    }

    public void setAddress1(String address1) {
        Address1 = address1;
    }

    public String getAddress2() {
        return Address2;
    }

    public void setAddress2(String address2) {
        Address2 = address2;
    }

    public String getSuburb() {
        return Suburb;
    }

    public void setSuburb(String suburb) {
        Suburb = suburb;
    }

    public String getPostCode() {
        return PostCode;
    }

    public void setPostCode(String postCode) {
        PostCode = postCode;
    }

    public String getState() {
        return State;
    }

    public void setState(String state) {
        State = state;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public String getHomePhoneNumber() {
        return HomePhoneNumber;
    }

    public void setHomePhoneNumber(String homePhoneNumber) {
        HomePhoneNumber = homePhoneNumber;
    }

    public String getEnable() {
        return Enable;
    }

    public void setEnable(String enable) {
        Enable = enable;
    }

    public String getPhoneNumber() {
        return PhoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        PhoneNumber = phoneNumber;
    }

    public UserAccount getUserAccount() {
        return userAccount;
    }

    public void setUserAccount(UserAccount userAccount) {
        this.userAccount = userAccount;
    }

    public String getLimit() {
        return Limit;
    }

    public void setLimit(String limit) {
        Limit = limit;
    }
}

File: iOS/Telehealth/Patient/Telehealth/Telehealth/Patient.swift
//
//  Patient.swift
//  Telehealth
//
//  Created by Giap Vo Duc on 10/27/15.
//  Copyright © 2015 Giap Vo Duc. All rights reserved.
//

import Foundation

class Patient{
    
    var MiddleName : String!
    var Address2 : String!
    var Title : String!
    var WorkPhoneNumber: String!
    var Enable : String!
    var PhoneNumber : String!
    var Occupation : String!
    var LastName : String!
    var Postcode : String!
    var UID : String!
    var UserAccountID : String!
    var Gender : String!
    var FirstName : String!
    var State : String!
    var ModifiedDate : String!
    var Email1 : String!
    var Country : String!
    var ID : String!
    var Address1 : String!
    var CountryID : String!
    var DOB : String!
    var Suburb : String!
    var HomePhoneNumber : String!
    var Image : UIImage!
    init(MiddleName:String,Address2:String,Title:String,WorkPhoneNumber:String,Enable:String,PhoneNumber:String,Occupation:String,LastName:String,Postcode:String,UID:String,UserAccountID:String,Gender:String,FirstName:String,State:String,ModifiedDate:String,Email1:String,Country:String,ID:String,Address1:String,CountryID:String,DOB:String,Suburb:String,HomePhoneNumber:String,Image:UIImage){
        
        self.MiddleName = MiddleName
        self.Address2 = Address2
        self.Title = Title
        self.WorkPhoneNumber = WorkPhoneNumber
        self.Enable = Enable
        self.PhoneNumber = PhoneNumber
        self.Occupation = Occupation
        self.LastName = LastName
        self.Postcode = Postcode
        self.UID = UID
        self.UserAccountID = UserAccountID
        self.Gender = Gender
        self.FirstName = FirstName
        self.State = State
        self.ModifiedDate = ModifiedDate
        self.Email1 = Email1
        self.Country = Country
        self.DOB = DOB
        self.Suburb = Suburb
        self.HomePhoneNumber = HomePhoneNumber
        self.Address1 = Address1
        self.Image = Image
        
    }
    
    func getAPI(){
        
    }

    
    
    
}